superset/views/core.py

```diff
@@ -905,6 +908,7 @@ def explore(
             "user": bootstrap_user_data(g.user, include_perms=True),
             "forced_height": request.args.get("height"),
             "common": common_bootstrap_payload(),
+            "columns": columns,
```

Comment:
The current columns are passed in to the datasource object. For easier lookup, do you think we can keep the same structure for queries, instead of having to look in two different places?